util/data_object.py

```diff
@@ -202,3 +209,13 @@ def decode_checksum(checksum):
         return "0"
     else:
         return binascii.hexlify(binascii.a2b_base64(checksum[5:])).decode("UTF-8")
+
+
+def get_groups(ctx, data_id):
```

Comment:
Can you add an integration test for this function to the data object integration tests, please? 

Existing integration tests:
https://github.com/UtrechtUniversity/yoda-ruleset/blob/1886415e0b17c6e96cbf4d279a2364544f873d7c/integration_tests.py#L60-L72

I feel it would be useful if the function name reflects what group role it is searching for. Perhaps something like `get_group_owners` or `get_groups_with_own_permission`?